lighthouse-core/config/default.js

```diff
@@ -224,6 +224,19 @@ module.exports = {
           '[PWA Checklist](https://developers.google.com/web/progressive-web-apps/checklist) but are ' +
           'not automatically checked by Lighthouse. They do not affect your score but it\'s important that you verify them manually.',
     },
+    'seo-mobile': {
+      title: 'Mobile Friendly',
+      description: 'Make sure your pages are mobile friendly so users don’t have to pinch or zoom ' +
+          'in order to read the content pages. [Learn more](https://developers.google.com/search/mobile-sites/).',
+    },
+    'seo-content': {
+      title: 'Content Best Practices',
+      description: 'Format your HTML in a way that enables crawlers to better understand your app’s content.',
+    },
+    'seo-crawl': {
+      title: 'Crawling and Indexing',
+      description: 'To appear in search results, crawlers need access to your app.',
```

Comment:
Hm… I wonder why these groups are not shown in the report as they are in e.g. accessibility:

<img width="528" alt="screen shot 2017-12-14 at 19 56 39" src="https://user-images.githubusercontent.com/985504/34009373-ee3c3fd6-e108-11e7-8148-5b93bb4ba7ff.png">

<img width="442" alt="screen shot 2017-12-14 at 19 56 31" src="https://user-images.githubusercontent.com/985504/34009380-f55deef4-e108-11e7-940c-b7fe6a14dc6c.png">